### DISCUSSION TITLE
'Allow custom backgrounds' does not work

### DISCUSSION THREAD

**Discussion by allesoverwaterstof** [JulianPrieber/littlelink-custom#336] [Issues]
This option on the admin page does not work:
Allow custom backgrounds
Allow users to upload custom background images for their pages.

When enabled, users still do not see this option on their page settings.

**Comment by JulianPrieber** [comment]
So um, I'm majorly confused right now.
I checked this option and there indeed seems to be a problem.

However, I'm seeing the exact opposite of what you're describing here?

The option is enabled by default, so you have to disable it manually after the update to disable it. And this disabling function doesn't seem to work properly.

**Reply by JulianPrieber** [reply]
It seems the wrong version was pushed to release. I'm going to fix this with a patch shortly.

The thing is this means **in the current latest build it is not possible at all to disable this feature.**
It's always on by default and everyone can use it.

**Reply by allesoverwaterstof** [reply]
Well I don't have that option on the site of a user, only on the admin pages

**Reply by JulianPrieber** [reply]
Ahhh, I see now. Can confirm.

We seem to have missed this in the beta test.
This will be fixed in the patch too.

**Reply by allesoverwaterstof** [reply]
thanks!

**Comment by allesoverwaterstof** [comment]
Changing custom backgrounds still gives troubles. When I delete an old one and upload a new one, the old one still is shown.
And on the admin page now the 'logo' that a user uses is shown very very large... Just trying to help out. I love your project!

**Reply by streamerstv** [reply]
I am also having this problem! I can't delete the old background
